agent.go

```diff
@@ -24,12 +24,14 @@ import (
 	"time"
 
 	"github.com/gogo/protobuf/proto"
+	"github.com/grpc-ecosystem/grpc-opentracing/go/otgrpc"
```

Comment:
@jodh-intel otgrpc has `ocgrpc.DefaultServerViews` those capture all gRPC calls. If you use them you get a lot of the tracing for free at the gPRC level w/o instrumenting the actual callbacks.

Reply:
Sorry I meant ocgrpc has it. otgrpc does not seem to have an equivalent